src/client/package.json

```diff
@@ -56,7 +56,7 @@
     "e2e:web:smoke": "playwright test --project=chrome --grep @smoke"
   },
   "dependencies": {
-    "@adaptive/hydra-platform": "^3.120.0",
+    "@adaptive/hy‰dra-platform": "3.120.0",
```

Comment:
There is a weird character after y in `hydra`, see picture below:

Reply:
![image](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/assets/10324761/fd513cd9-05fe-4cda-ae01-54b832fa09ac)

Reply:
y, I saw that .. should have pushed up correction last week, sorry for the distraction